### ISSUE TITLE
Dependencies

### ISSUE THREAD

**Issue by Niggler** [epfromer/pst-extractor#1]
First off, awesome library!

Looking at the dependencies, some are superfluous and others can be avoided:

- `rimraf` is not used at all

- `winston-loggly-bulk` is commented out and `winston-loggly` is not used

- `fs-ext` you can keep track of the file position explicitly in `PSTFile`, since the library never does a seek from the end of the file.  This would also avoid the native node module.

- `math-float64-from-bits` you can use a typed array for this:

```js
// interpret byte array as double
var b = [205, 204, 204, 204, 204, 220, 94, 64]; new Float64Array(new Uint8Array(b).buffer)[0]
// interpret {low, high} signed 32 bit integers as double
var o = {low:0xcccccccd, high:0x405edccc}; new Float64Array(new Int32Array([o.low, o.high]).buffer)[0]
```

**Comment by epfromer** [comment]
I just published v1.3.0 which incorporates all your feedback.

**Comment by Niggler** [comment]
`rimraf` is still in the package.json for `1.3.0`.  It appears to only be called in the `prebuild` script, which iirc is not part of the `install` lifecycle.  Shouldn't it be a dev dep instead of a normal dependency?

**Comment by epfromer** [comment]
Good point - just fixed and issued v1.3.1.

**Comment by Niggler** [comment]
Looks good 👍
